Classes/Format/Alto.php

```diff
@@ -42,4 +46,49 @@ public function getRawText(\SimpleXMLElement $xml)
         }
         return $rawText;
     }
+
+
+
+    /**
+     * This extracts the fulltext data from ALTO XML and returns it in MiniOCR format
+     *
+     * @access public
+     *
+     * @param \SimpleXMLElement $xml: The XML to extract the raw text from
+     *
+     * @return string The unformatted fulltext in MiniOCR format
+     */
+    public function getTextAsMiniOcr(\SimpleXMLElement $xml)
+    {
+        $rawText = '';
```

Comment:
There is a complain from Scrutinizer about not used `$rawText` variable. It needs to to be removed or used in return in case blocks are empty.

Reply:
Thank you. That's fixed now. This was taken from the `getRawText()` method, I've modified before.